doc/mlsys_paper/paper.tex

```diff
@@ -418,14 +418,17 @@ \section{Example: Learning Linear Regression Models}
 ensmallen} provides.
 
 \vspace*{-0.3em}
-\section{Available optimizers}
+\section{Available Optimizers}
 \vspace*{-0.5em}
 
-Due to the easy abstraction, {\tt ensmallen} is able to provide a large set of
-diverse optimization algorithms.  Below is a list of what is currently
-available.
+Due to the straightforward abstraction framework, {\tt ensmallen} is able to
+provide a large set of diverse optimization algorithms.  Below is a list of
+currently available optimizers:
 
 % I guess to save some space we should group them.
+%% CS: should we add citations to some (or all) of the listed optimisers?
+%% CS: the MLSYS CFP states that "papers can be up to 6 pages (not including references)",
+%% CS: so technically we can have 10000+ references.
```

Comment:
I see no issue with that.  I'll start gathering references and add them tonight.